### PR TITLE
Simplify tp+pp example by using pippy.all_compile

### PR DESCRIPTION
**Pull request by kwen2501**
`pippy.all_compile` would output the stage module, hence easier for TP.

### CODE REVIEW

**Comment by kwen2501** [comment]
Cc @fduwjj @wanchaol

File: examples/tp+pp/pippy_tp.py [hunk 1]
```diff
@@ -180,8 +158,7 @@ def main(args=None):
         "-s",
         "--schedule",
         type=str,
-        default=list(schedules.keys())[0],
-        choices=schedules.keys(),
+        default="FillDrain",
```
Comment (fduwjj) [inline examples/tp+pp/pippy_tp.py:161]: N00b question what does this mean?

**Review by fduwjj — APPROVED** [review]
LGTM

File: examples/tp+pp/pippy_tp.py (hunk 1 above)
Comment (kwen2501) [reply examples/tp+pp/pippy_tp.py:161]: Fill-drain is a pipeline scheduling policy. Other options include 1F1B, interleaved 1F1B. See for example [this paper](https://arxiv.org/pdf/2104.04473.pdf), section 2.2.

**Pull request merged**